packages/core/test/test-util/test-services/v4/test-service/function-imports.ts

```diff
@@ -87,6 +87,46 @@ export function testFunctionImportEntityReturnTypeCollection(parameters: TestFun
   return new FunctionImportRequestBuilderV4('/sap/opu/odata/sap/API_TEST_SRV', 'TestFunctionImportEntityReturnTypeCollection', (data) => transformReturnValueForEntityListV4(data, TestEntity), params);
 }
 
+/**
+ * Type of the parameters to be passed to [[testFunctionImportSharedEntityReturnType]].
+ */
+export interface TestFunctionImportSharedEntityReturnTypeParameters {
+}
+
+/**
+ * Test Function Import Shared Entity Return Type.
+ *
+ * @param parameters - Object containing all parameters for the function import.
+ * @returns A request builder that allows to overwrite some of the values and execute the resultng request.
+ */
+export function testFunctionImportSharedEntityReturnType(parameters: TestFunctionImportSharedEntityReturnTypeParameters): FunctionImportRequestBuilderV4<TestFunctionImportSharedEntityReturnTypeParameters, any> {
```

Comment:
See comment below
```suggestion
export function testFunctionImportSharedEntityReturnType(parameters: TestFunctionImportSharedEntityReturnTypeParameters): Exclude<FunctionImportRequestBuilderV4<TestFunctionImportSharedEntityReturnTypeParameters, any>, 'execute> {
```

Reply:
As discussed, this will be the chosen API design.